$NetBSD: patch-apps_tqsl_cpp,v 1.3 2015/10/11 02:59:07 mef Exp $

(1)
Avoid following error (interim, only valid with db5)
/usr/pkgsrc/wip/trustedQSL/work/tqsl-2.0/apps/tqsl.cpp:1133:36: error: 'DB_VERSION_STRING' was not declared in this scope
gmake[2]: *** [apps/CMakeFiles/tqsl.dir/tqsl.cpp.o] Error 1
gmake[1]: *** [apps/CMakeFiles/tqsl.dir/all] Error 2

(2)
tqsl-2.0.1/apps/tqsl.cpp:2963:10: error: #pragma GCC diagnostic not allowed inside functions
tqsl-2.0.1/apps/tqsl.cpp:2967:10: error: #pragma GCC diagnostic not allowed inside functions

--- apps/tqsl.cpp.orig	2015-12-24 11:24:19.000000000 +0900
+++ apps/tqsl.cpp	2015-12-31 00:58:28.000000000 +0900
@@ -64,7 +64,7 @@
 #endif
 #include <zlib.h>
 #include <openssl/opensslv.h> // only for version info!
-#include <db.h> //only for version info!
+#include <db5/db.h> //only for version info!
 
 #include <iostream>
 #include <fstream>
@@ -1725,7 +1725,8 @@ MyFrame::EditQSOData(wxCommandEvent& WXU
 	tqslTrace("MyFrame::EditQSOData");
 	QSORecordList recs;
 	wxString file = wxFileSelector(_("Open File"), wxConfig::Get()->Read(wxT("QSODataPath"), wxT("")), wxT(""), wxT("adi"),
-#if !defined(__APPLE__) && !defined(_WIN32)
+#if !defined(__APPLE__) && !defined(_WIN32) && !defined(__NetBSD__) && !defined(__DragonFly__) && !defined(__FreeBSD__)
+
 			_("ADIF files (*.adi;*.adif;*.ADI;*.ADIF)|*.adi;*.adif;*.ADI;*.ADIF|All files (*.*)|*.*"),
 #else
 			_("ADIF files (*.adi;*.adif)|*.adi;*.adif|All files (*.*)|*.*"),
